src/markdown_parser.py

```diff
@@ -0,0 +1,43 @@
+import re
+from html import escape, unescape
+from typing import Match
+
+import mistune  # type: ignore
+
+# https://gist.github.com/gruber/8891611
+URL_REGEX = r"""(?i)([^"\>\<\/\.]|^)\b((?:https?:(/{1,3}))(?:[^\s()<>{}\[\]]+|\([^\s()]*?\([^\s()]+\)[^\s()]*?\)|\([^\s]+?\))+(?:\([^\s()]*?\([^\s()]+\)[^\s()]*?\)|\([^\s]+?\)|[^\s`!()\[\]{};:'".,<>?«»“”‘’]))"""
+
+
+class GithubToAsanaRenderer(mistune.HTMLRenderer):
+    def paragraph(self, text):
+        return text + "\n"
+
+    def block_quote(self, text):
+        return f"<em>> {escape(text, quote=False)}</em>"
```

Comment:
@owenauch  - I think this `<em>>` is causing SGTM to break - I assume this extra `>` was not intentional right?

see: https://app.asana.com/0/0/1200718797414710/1200725140636985/f

Reply:
Nevermind -- I think https://github.com/Asana/SGTM/pull/101/files should fix it